styles.css

```diff
@@ -0,0 +1,333 @@
+    color:rgb(97, 97, 97);
+    font-size: 1.6rem;
+    
+}
+.click{
+    color:rgb(97, 97, 97);
+    font-size: 1.5rem;
+    margin-left: 0px;
+    margin-top: 4rem;
+    text-decoration: underline;
+}
+.secondhalf{
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    margin-top: 15rem;
+    padding-top: 5rem;
+    padding-bottom: 5rem;
+    padding-left: 4rem;
+    padding-right: 4rem;
+    width: 100%; 
+    background-color: rgb(36, 35, 33, .6); 
+        
+}
+
+.front{
+    
+    width: 100%;
+    background-color: rgb(46, 46, 46); 
+    display: flex;
+    flex-wrap: wrap;
+    align-items: center;
+    background-image:url(asset/cameraImage.svg);
+    background-position: center;
+    
+    
+ }
+  .leftside{
+    border-right: 1px solid rgb(255, 255, 255);
+    padding: 5rem;
+    flex-grow: 1;
+    flex-basis: 50%;
+  }
+  
+ .fill{
+    color: rgb(255, 255, 255);
+    font-size:1.5rem;
+    margin-top: 2rem;
+    margin-bottom: 2rem;
+    
+ }
+
+ .form input[type="text"]
+ {
+    color: rgb(255, 255, 255);
+    border: none;
+    outline: none;
+    font-size:1.5rem;
+    padding: auto;
+    border-bottom: 1px solid white;
+    background-color: transparent;
+    margin-top: 3rem;
+ }
+ .form input[type="email"]
+ {
+    color: rgb(255, 255, 255);
+    border: none;
+    outline: none;
+    font-size:1.5rem;
+    border-bottom: 1px solid rgb(255, 255, 255);
+    background-color: transparent;
+    margin-top: 3rem;
+ }
+ .form input[type="number"]
+ {
+    color: rgb(255, 255, 255);
+    border: none;
+    outline: none;
+    font-size:1.5rem;
+    border-bottom: 1px solid rgb(255, 255, 255);
+    background-color: transparent;
+    margin-top: 3rem;
+ }
+ .city{
+    color: gray;
+    border: none;
+    outline: none;
+    border-bottom: 1px solid rgb(255, 255, 255);
+    background-color: transparent;
+    margin-top: 3rem;
+    font-size:1.5rem;
+ }
+ #fullname{
+    width: 50%;
+ }
+ #email{
+    width: 50%;
+ }
+ #phone{
+    width: 50%;
+ }
+ #street{
+    width: 50%;
+ }
+ #street2{
+    width: 50%;
+ }
+ #zip{
+    width:15%;
+ }
+ .state{
+    color: gray;
+    border: none;
+    outline: none;
+    border-bottom: 1px solid rgb(255, 255, 255);
+    background-color: transparent;
+    margin-top: 3rem;
+    font-size:1.5rem;
+    margin-left: 3rem;
+ }
+ .country{
+    color: gray;
+    border: none;
+    outline: none;
+    border-bottom: 1px solid rgb(255, 255, 255);
+    background-color: transparent;
+    margin-top: 3rem;
+    font-size:1.5rem;
+ }
+ .zip{
+    margin-left: 3rem;
+ }
+ .gender{
+    font-size:1.5rem;
+    margin-top: 3rem;
+    color: rgb(97, 97, 97);
+ }
+ .rightside{
+    flex-grow: 1;
+    flex-basis: 50%;
+    padding-left: 4rem;
+ }
+ 
+ .description{
+    color: rgb(255, 255, 255);
+    font-size: 1.5rem;
+    margin-top: 6rem;
+      
+ }
+ .destext{
+    background-color: transparent;
+    border-radius: 20px;
+    width: 50%;
+    resize: none;
+    height: 15vh;
```

Comment:
use min-height

Reply:
using min-height property, the line which is separating the second form (fill the application) is not touching its two ends.